<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppDialog</class>
 <widget class="QDialog" name="AppDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome to Your Budgeting App!</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>541</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>My Balance</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>501</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string>Edit Balance</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Current Balance</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>151</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Food</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_foodCost">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Rent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_rentCost">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>151</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Entmt.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_entertainmentCost">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Tuition</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_tuitionCost">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Savings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_savingsCost">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_24">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Misc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_miscCost">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>119</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount to Change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Deposit or Withdraw?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_depositFoodC">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawFoodC">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_depositRentC">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawRentC">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QPushButton" name="pushButton_depositEntertainmentC">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawEntertainmentC">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QPushButton" name="pushButton_depositTuitionC">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawTuitionC">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <item>
        <widget class="QPushButton" name="pushButton_depositSavingsC">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawSavingsC">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_25">
       <item>
        <widget class="QPushButton" name="pushButton_depositMiscC">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawMiscC">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>My Budget</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>501</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Edit Budgets</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_14">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Current Budget</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>151</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Food</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_foodBudget">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Rent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_rentBudget">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>151</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Entmt.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_entertainmentBudget">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Tuition</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_tuitionBudget">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Savings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_savingsBudget">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_26">
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Misc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_miscBudget">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>119</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount to Change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Add or Subtract?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_21">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="pushButton_depositFoodB">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawFoodB">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_22">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QPushButton" name="pushButton_depositRentB">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawRentB">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_23">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QPushButton" name="pushButton_depositRentE">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawEntertainmentB">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_24">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QPushButton" name="pushButton_depositTuitionB">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawTuitionB">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_25">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_27">
       <item>
        <widget class="QPushButton" name="pushButton_depositSavingsB">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawSavingsB">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_26">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_28">
       <item>
        <widget class="QPushButton" name="pushButton_depositMiscB">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_withdrawMiscB">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_signOut">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_getUsername">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
